src/PrestaShopBundle/Resources/views/Admin/macros.html.twig

```diff
@@ -163,13 +163,16 @@
           {% endset %}
       {% endif %}
 
+      <template class="js-popover-error-content" data-id="{{ form.vars.id }}">
```

Comment:
I did not know `template` keyword, what is its behavior ? I cant find it in https://twig.symfony.com/doc/

Reply:
its perfect for holding the content which only is being used in javascript. https://www.w3schools.com/tags/tag_template.asp . Its same as div but I think its purpose is to not display the content for the client untill it is needed :)